doc/latest_news.html

```diff
@@ -4,6 +4,7 @@
       December 22nd 2015: HoloViews 1.4.1 released and now available on 
       <a href="https://pypi.python.org/pypi/holoviews/">PyPI</a> and
       <a href="https://anaconda.org/anaconda/holoviews">Anaconda</a>.
+      Now includes full support for the <a href="http://bokeh.pydata.org">Bokeh</a> plotting library.
```

Comment:
Regarding cross-backend layouts, when Philipp and I were discussing that, I briefly thought that PhosphorJS coupled with some custom stitching of SVG into larger SVGs might make us be able to solve layout problems once, for all backends.  But it's hard to see how we could do that when considering aligning plots by their active area (main plot), not just their bounding boxes (including axes, labels, etc.)  *Maybe* we could extract that information from all backends and supply it to single layout engine, but that seems ambitious.  So even if we do have some support for cross-backend layouts, it seems unlikely we'll be able to use that for everything...